### ISSUE TITLE
Documentation is not consistent for the CLI arguments

### ISSUE THREAD

**Issue by Lawouach** [containous/traefik#5424]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

I read the documentation.

### What did you expect to see?

Consistency.

### What did you see instead?

Differences in CLI arguments.

In many places examples of uising the CLI use a different case for their arguments.
For example, https://docs.traefik.io/https/acme/#dnschallenge says:

```
--certificatesResolvers.sample.acme.httpChallenge.entryPoint=web
```

But then the main page for all the arguments (https://docs.traefik.io/reference/static-configuration/cli/) says:

```
--certificatesresolvers.<name>.acme.httpchallenge.entrypoint
```

All lower case. Is there a semantic difference or do they both work? The logs don't say anything.

I'm using 2.0 GA from Kubernetes via the CRD.

**Comment by ldez** [comment]
Hello,

The flags names are case insensitive, there is not semantic difference.

**Comment by Lawouach** [comment]
It is confusing to me but if that's what it is, then cool. I must have missed the documentation clarifying it.

**Comment by traefiker** [comment]
Closed by #5428.

**Comment by Lawouach** [comment]
Cheers folks!
